viral/basic_plots.py

```
@@ -0,0 +1,766 @@
+                save_path = (
+                    SERVER_PATH
+                    / "viral_caches"
+                    / "firing_rates"
+                    / f"{mouse_name}_{date}_rewarded_{rewarded}_firing_rates.npy"
+                )
+                # Weird suffix fiddle because the file is saved as .npy.npz
+                load_path = save_path.with_suffix(save_path.suffix + ".npz")
+                if load_path.exists():
+                    print(f"aready done {mouse_name}, {date}")
+                    result_mouse = np.load(load_path)
+                    result[stage].append(
```

Ruff (0.14.4)

Undefined name `result`

(F821)